Add arrangement management api page (#29)
wso2/docs-ob-toolkit-cds: en/docs/learn/pushed-authorization-endpoint.md

> The pushed authorization request endpoint is an HTTP API at the authorization server that accepts HTTP "POST" requests with

Reviewer: ```suggestion
The Pushed Authorization Request (PAR) endpoint is an HTTP API at the authorization server. This endpoint accepts **HTTP POST** requests with
```

Reviewer: The CDS spec use 'z'

Reviewer: The first occurence of Pushed Auth Req we need to use the Abbreviation, otherwise we cannot use it throughout the doc.

> parameters in the HTTP request message body using the "application/x- www-form-urlencoded" format.

Reviewer: ```suggestion
parameters in the HTTP request message body using the `application/x- www-form-urlencoded` format.
```

Reviewer: In doc space we do not use double quotes. Instead let's bold them (make them italics/change the formatting)

> Unlike in the Authorisation endpoint, in the Pushed Authorisation endpoint, Data Recipients pushes authorisation 

Reviewer: ```suggestion
Unlike in the Authorization endpoint, in the Pushed Authorization endpoint, Data Recipients pushes authorisation 
```

Reviewer: The spec is using 'z'

> details directly to the authorisation server and obtains a reference. The Data Recipients obtain `request_uri` which is a reference 

Reviewer: ```suggestion
details directly to the authorisation server and obtain a reference. The Data Recipients obtain `request_uri`, which is a reference 
```

Reviewer: Since we are referring to Data Recipients, we can use  'obtain'

to the authentication and authorization details sent in the pushed authorization request. Thereby, it prevents:
 
   - Intruders from intercepting the authorisation information sent in the `request_object`.
   - Authorisation request calls becoming large with the authorisation details signed in the JWT.

> ###PAR endpoint

Reviewer: ```suggestion
##PAR endpoint
```

Upon successful invocation of the `/par` endpoint, Data Recipients will receive a `request_uri` value with an expiration time.
Therefore, the reference is only valid until the expiration time for the subsequent authorization invocation.

> Given below is a sample request and a response:

Reviewer: ```suggestion
Given below are sample request and response:
```

```

``` tab="Response"
{
    "request_uri": "urn:ietf:params:oauth:request_uri:bwc4JK-ESC0w8acc191e-Y1LTC2",
    "expires_in": 60
}
```

This same `request_uri` value is used in the subsequent authorization request as well.

??? tip "Click here to see configurations related to the Pushed Authorization web application..."
     1. Open the `<IS_HOME>/repository/conf/deployment.toml` file.
     2. Add the following configurations that allow you to change the format and the expiration time of the `request_uri` reference:

          ```
          [open_banking.push_authorisation]
          expiry_time=60
          request_uri_sub_string="substring"
          ```

        !!! note
              You can change the format of the request_uri using the `request_uri_sub_string` tag.
              
              ```
              {
                  "request_uri": "urn:<substring>:bwc4JK-ESC0w8acc191e-Y1LTC2",
                  "expires_in": 60
              }
              ```

###Request URI

> The `request_uri` corresponding to the authorization request posted. This URI is a single-use reference to the respective request data

Reviewer: ```suggestion
The `request_uri` corresponds to the authorization request posted. This URI is a single-use reference to the respective request data
```
